targetplatform/smarthome.target

```diff
@@ -101,6 +101,10 @@
 <repository location="http://download.eclipse.org/modeling/emf/emf/updates/2.11/core/R201506010402"/>
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.2.0.v20170518-0647"/>
```

Comment:
Is this an incubation project? If so, we might require a CQ for it, right?

Reply:
The due diligence process allows "Re-Use of Eclipse Project or Plug-in (Orbit excluded) or bundle that has undergone a release review". It doesn't say anything about incubation status of Eclipse projects. LSP4J has been [formally released](https://projects.eclipse.org/projects/technology.lsp4j/reviews/0.2.0-release-review). 
Did I miss anything?

Reply:
Ok, then we are lucky (and I had something wrong in mind... reminder to self: incubation!=unreleased).